python/tvm/relax/frontend/legalizer.py

```diff
@@ -0,0 +1,136 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import math
+
+import tvm
+from tvm import relax, te, topi
+from tvm.ir.module import IRModule
+from tvm.relax.block_builder import BlockBuilder
+
+
+@relax.expr_functor.mutator
+class OperatorLegalizer(relax.PyExprMutator):
```

Comment:
This should not be part of front-end, but can be part of relax.transform.

Reply:
Moved